.travis.mk

```diff
@@ -282,6 +282,7 @@ test_static_build_cmake_linux: deps_tests
 
```

Comment:
> NO_DOC=cmake
> NO_CHANGELOG=cmake
> NO_TEST=cmake

cmake?

Reply:
yes, cmake, as in https://github.com/tarantool/tarantool/commit/07b103832f0002c78cb77b715d861b7d876f7f45

Reply:
replaced with "ci"